client/components/Prereqs/CSEPrereq.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import Flowchart from 'react-simple-flowchart';
+import Head from '../Header-Footer/Head';
+import Foot from '../Header-Footer/Foot';
+import { Menu, Dropdown, Button } from 'antd';
+
+
+class CSEPrereq extends React.Component {
+
+    constructor(props) {
+        super(props);
+        const code =
+            `
+COP3502=>condition: COP3502|department1:>http://localhost:8000/#/ClassDetails/CS/COP%203502
+COP3503=>condition: COP3503|department2:>http://localhost:8000/#/ClassDetails/CS/COP%203503
+COT3100=>condition: COT3100|department3:>http://localhost:8000/#/ClassDetails/CS/COT%203100
+EEL3701C=>condition: EEL3701C|department4:>http://localhost:8000/#/ClassDetails/ECE/EEL%203701
+COT4501=>condition: COT4501|department5:>http://localhost:8000/#/ClassDetails/CS/COT%204501
+CDA3101=>condition: CDA3101|department6:>http://localhost:8000/#/ClassDetails/CS/CDA%203101
+COP3530=>condition: COP3530|department7:>http://localhost:8000/#/ClassDetails/CS/COP%203530
+CIS4301=>condition: CIS4301|department8:>http://localhost:8000/#/ClassDetails/CS/CIS%204301
+CEN3031=>condition: CEN3031|department9:>http://localhost:8000/#/ClassDetails/CS/CEN%203031
+COP4600=>condition: COP4600|department10:>http://localhost:8000/#/ClassDetails/CS/COP%204600
+CNT4007C=>condition: CNT4007C|department11:>http://localhost:8000/#/ClassDetails/CS/CNT%204007
```

Comment:
for CNT4007C, the color of dark blue very hard to see. maybe consider using another color